tools/travis/build.sh

```diff
@@ -37,7 +37,7 @@ export OPENWHISK_HOME=$WHISKDIR
 
```

Comment:
@tareqmamari We do not install the catalog here by $ANSIBLE_CMD postdeploy.yml, since we will install and test and catalog from this repo, not the one from openwhisk/catalog.